### DISCUSSION TITLE
requested module '/node_modules/.vite/deps/vue-router.js?v=85d94ed7' does not provide an export named 'default'

### DISCUSSION THREAD

**Discussion by hulzzz** [vuejs/router#1952] [Help and Questions] [answered]
Using vue for the first time, how to solve this problem

**Comment by posva** [comment, marked as answer]
```ts
import { createRouter } from 'vue-router'
```

There is no longer a default export in Vue Router but the docs are still showing a CDN approach: https://github.com/vuejs/router/issues/829

**Reply by posva** [reply]
BTW, you can start of with something like https://github.com/antfu/vitesse-lite if you want a scaffolded project that just works
